Source/SA/RFmx NR/RFmxNR.csproj

```diff
@@ -18,7 +18,7 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\..\..\..\..\..\..\..\bin\Debug\</OutputPath>

```

Comment:
Same comment as in DPD. What's the story behind these crazy output paths for the debug dll?

Reply:
See above.